src/mlagility/analysis/analysis.py

```diff
@@ -139,17 +140,29 @@ def call_benchit(
         else:
             model_info.status_message = "Build Error: see log files for details."
         model_info.status_message_color = printing.Colors.WARNING
+        exc_type, exc_value, exc_traceback = sys.exc_info()
+        model_info.traceback = traceback.format_exception(
+            exc_type, exc_value, exc_traceback
+        )
```

Comment:
This is reused 3 times. probably worth creating a function for that.

Reply:
Good suggestion, thanks!

Reply:
Good suggestion, done.